papaparse.js

```diff
@@ -1403,8 +1403,8 @@ License: MIT
 		var preview = config.preview;
 		var fastMode = config.fastMode;
 		var quoteChar;
-		/** Allows for no quoteChar by setting quoteChar to undefined in config */
-		if (config.quoteChar === undefined) {
+		/** Allows for no quoteChar by setting quoteChar to undefined or null in config */
```

Comment:
I think this is obvious, so the comment is not that useful.

```suggestion
```

Reply:
effectively, i agree with you.